server/graphql/v1/mutations/orders.js

```diff
@@ -119,6 +120,16 @@ async function checkRecaptcha(order, remoteUser, reqIp) {
   return response;
 }
 
+async function handleHostPlanLimit(host) {
```

Comment:
Maybe a more specific name `handleHostPlanAddedFundsLimit` and can you move that in `lib/plan` ?

Reply:
Ideally all similar code should be also extracted in the same way to `lib/plan`. This would help consolidate all the handling there. Great if we're doing it now abut that can also be done later.